xkcd_dl/annotated.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
```

Comment:
I also guessed so, since most of the logic is repeated here. Let's just remove this file. What do you say?